librz/core/cmd_descs/cmd_analysis.yaml

```diff
@@ -996,6 +996,157 @@ commands:
             type: RZ_CMD_ARG_TYPE_RZNUM
             optional: true
+  - name: ag
+    summary: Analysis graph commands
+    details:
+      - name: "Formats"
+        entries:
+          - text: "<blank>"
+            comment: "Ascii art"
+          - text: "*"
+            comment: "rizin commands"
+          - text: "d"
+            comment: "Graphviz dot"
+          - text: "g"
+            comment: "Graph Modelling Language (gml)"
+          - text: "j"
+            comment: "json ('J' for formatted disassembly)"
+          - text: "k"
+            comment: "SDB key-value"
+          - text: "t"
+            comment: "Tiny ascii art"
+          - text: "v"
+            comment: "Interactive ascii art"
+          - text: "w [path]"
+            comment: "Write to path or display graph image (see graph.gv.format)"
+    subcommands:
+      - name: aga
+        summary: Data reference graph
+        cname: analysis_graph_dataref
+        args:
+          - name: format
+            type: RZ_CMD_ARG_TYPE_STRING
+            optional: true
+      - name: agA
+        summary: Global data references graph
+        cname: analysis_graph_dataref_global
+        args:
+          - name: format
+            type: RZ_CMD_ARG_TYPE_STRING
+            optional: true
+      - name: agc
+        summary: Function callgraph
+        cname: analysis_graph_callgraph_function
+        args:
+          - name: format
+            type: RZ_CMD_ARG_TYPE_STRING
+            optional: true
+      - name: agC
+        summary: Global callgraph
+        cname: analysis_graph_callgraph_global
+        args:
+          - name: format
+            type: RZ_CMD_ARG_TYPE_STRING
+            optional: true
+      - name: agd
+        summary: Diff graph
+        cname: analysis_graph_diff
+        args:
+          - name: format
+            type: RZ_CMD_ARG_TYPE_STRING
+            optional: true
+          - name: addr
+            type: RZ_CMD_ARG_TYPE_RZNUM
+            optional: true
+      - name: agf
+        summary: Basic blocks function graph
+        cname: analysis_graph_bb_function
+        args:
+          - name: format
+            type: RZ_CMD_ARG_TYPE_STRING
+            optional: true
+      - name: agi
+        summary: Imports graph
+        cname: analysis_graph_imports
+        args:
+          - name: format
+            type: RZ_CMD_ARG_TYPE_STRING
+            optional: true
+      - name: agr
+        summary: References graph
+        cname: analysis_graph_refs
+        args:
+          - name: format
+            type: RZ_CMD_ARG_TYPE_STRING
+            optional: true
+      - name: agR
+        summary: Global references graph
+        cname: analysis_graph_refs_global
+        args:
+          - name: format
+            type: RZ_CMD_ARG_TYPE_STRING
+            optional: true
+      - name: ags
+        summary: Unknown graph
+        cname: analysis_graph_unknown
+        args:
+          - name: num
+            type: RZ_CMD_ARG_TYPE_NUM
```

Comment:
unknown graph?? be a bit clearer on what this does and what the argument is about

Reply:
I called it unknown because I have no idea what it does :D 
I will try to figure it out

Reply:
Maybe just normal graph, it just call `rz_core_analysis_graph(core, addr, 0)`, with empty `opts`